src/ocl_dev_mgr.cpp

```diff
@@ -4,365 +4,409 @@
 #include <fstream>
 #include <iostream>
 #include <iterator>
-
+#include <sstream>  
 #include "util.hpp"
 #include "ocl_dev_mgr.hpp"
 
 
 inline void compile(cl::Program& cl_prog, char const* options)
 {
-  std::string compile_options = std::string(" ") + std::string(options);
-
-  try {
-    cl_prog.build(compile_options.c_str());
-  }
-  catch (cl::BuildError error) {
-    std::string log = error.getBuildLog()[0].second;
-    std::cerr << ERROR_INFO << "Build error:\n" << log << std::endl;
-  }
-  catch (cl::Error err) {
-    std::cerr << ERROR_INFO << "Exception:" << err.what() << std::endl;
-  }
+	std::string compile_options = std::string(" ") + std::string(options);
```

Comment:
Did you use Visual Studio again? :stuck_out_tongue_winking_eye:

Reply:
😭